yt_dlp/extractor/svt.py

```diff
@@ -388,22 +386,63 @@ def _real_extract(self, url):
             dict_get(series, ('longDescription', 'shortDescription')))
 
 
-class SVTPageIE(InfoExtractor):
+class SVTPageIE(SVTBaseIE):
     _VALID_URL = r'https?://(?:www\.)?svt\.se/(?P<path>(?:[^/]+/)*(?P<id>[^/?&#]+))'
```

Comment:
we don't need to capture `path` anymore
```suggestion
    _VALID_URL = r'https?://(?:www\.)?svt\.se/(?:[^/?#]+/)*(?P<id>[^/?&#]+)'
```